content/blog/2018-04-23-noo-noo-from-collyers-college-wins-sourcebots-2018.md

```diff
@@ -0,0 +1,80 @@
+---
+title: “Noo Noo” from Collyer's College wins SourceBots 2018
+date: 2018-04-23
+---
+
+SourceBots 2018 reached its exciting climax this weekend as the teams from
+across the south coast gathered at the University of Southampton for the
+competition event. The two-day event saw their robots compete in forty matches
+to determine which would be crowned champion in this year's game
+_Pirate Islands_.
+
+<!--more-->
+
+### Competition
+
+The game required the robots to remove opposing tokens from their scoring zone
+and collect their own tokens from the opposing side of the 8×8m arena. As any of
+the four robots in the arena could move any token, this required the robots to
+detect which tokens were where and to react to changes.
+
+The matches were broken into two main sections. First was a league stage with
+matches spaced throughout the weekend and interspersed with "tinker time" which
+both allowed the teams to test and improve their robots as well as gauge the
+strengths of their opponents. The time between rounds proved invaluable to many
+of the teams and we saw the robots improve greatly over the course of the
+weekend.
+
+Once the league was complete, the results from the league seeded a knockout
+stage. While every robot entered the knockouts, it all came down to the final
+match between _[Noo Noo][CLY-robot-image]_ from [Collyer's][collyers] team
+_SCC_, _[Out of the blue][RGS-robot-image]_ from [RGS Guildford][rgs-guidlford],
+_[Brenda][BPV2-robot-image]_ from [Barton Pevril][barton-pevril]'s second team
+and [the robot][BRK-robot-image] from team _Bodgenhurst_ from
+[Brokenhurst][brokenhurst].
+
+[barton-pevril]: https://www.barton-peveril.ac.uk/
+[brokenhurst]: https://www.brock.ac.uk/
+[collyers]: http://www.collyers.ac.uk/
+[rgs-guidlford]: https://www.rgs-guildford.co.uk/
+
+[BPV2-robot-image]: /img/robots-2018/BPV2.jpg
+[BRK-robot-image]: /img/robots-2018/BRK.jpg
+[CLY-robot-image]: /img/robots-2018/CLY.jpg
+[RGS-robot-image]: /img/robots-2018/RGS.jpg
+
+The final promised to be interesting as each of the robots had fairly different
```

Comment:
The latter half this sentence feels strange to me. How about something like "The final promised to be interesting as a variety of different approaches were taken by the participating robots"?

Reply:
Yup, that's better; c810df51643f8f0e31b16f1d7ae37d5071a0884f